models/userExtension.js

```diff
@@ -29,11 +28,6 @@ module.exports = {
       type: Boolean,
       default: false,
     },
-    removedFromHomeScreen: Array,
-    improvementProjects: {
-      type: Array,
-      default: [],
-    },
     platformRoles: Array,
```

Comment:
@borkarsaish65 just make sure with @aks30  we can keep platformRoles key instead "programRoleMapping" as he suggested in the example during the review with him